app/src/main/java/org/gdg/frisbee/android/fragment/NewsFragment.java

```diff
@@ -64,10 +69,10 @@
 
     private NewsAdapter mAdapter;
 
-    public static NewsFragment newInstance(String plusId) {
+    public static NewsFragment newInstance(final String plusId) {

```

Comment:
The benefit is for other future developers I think. If someone tries to make an operation with plusId here, compiler will tell them not to do. But you are right, it is almost pointless for small functions.